portfolio/src/main/webapp/script.js

```diff
@@ -83,3 +85,28 @@ window.onload = (event) => {
     }
   })
 }
+
+function drawChart(commentInfo, numberOfComments){
+  fetch('/comment-'+commentInfo+'-data?numberOfComments='+String(numberOfComments)).then(response => response.json()).then((commentData) => {
+  trueCommentData = [];
+  for(letter in commentData){
+    if(letter == " "){
+       trueCommentData = trueCommentData.concat([["<SPACE>",commentData[letter]]]);
+    }
+    else{
+       trueCommentData = trueCommentData.concat([[letter,commentData[letter]]]);
+    }
```

Comment:
It looks prettier if you make indentation consistent:

```
for () {
  if () {
  } else {
  } 
}
```

Reply:
Everything is 2 spaces over from the previous, do you want 4?

Reply:
no; 2 space is fine. Consistency is important though.

Reply:
Can you tell me the exact line that is inconsistent because I am counting (starting on the line of the for loop and ending at the closing bracket):  2, 4, 6, 4, 4, 6, 4, 2.